packages/expo-localization/CHANGELOG.md

```diff
@@ -7,9 +7,13 @@
 ### 🎉 New features
```

Comment:
^ add a breaking change entry that explains that `locale` and `locales` return IETF BCP 47 locale identifiers, rather than platform-specific formats.

Reply:
I'm not sure this is needed. The behaviour hasn't really changed as the JavaScript level, which already returned hyphened locales (and still does).

Reply:
Oh, I see `parseLocale` already took care of this. Never mind, then. (Do we still need the parseLocale function?)

Reply:
No we don't. I've checked all platforms and all return hyphens. I've removed it 👍